lib/medium-admin.php

```diff
@@ -136,6 +136,12 @@ public static function add_meta_boxes_post($post) {
   public static function save_post($post_id, $post) {
     if (defined('DOING_AUTOSAVE') && DOING_AUTOSAVE) return;
 
+    $allowed_post_types = (array) apply_filters("medium_allowed_post_types", array("post"));
```

Comment:
@majelbstoat Hmmm OK, I happen to think type casting output on filters to be a best practice :smile: 

In this case it prevents unexpected filter returns from causing a PHP warning during `in_array()`. It would also allow a developer to just return a string.

```php
print_r( (array) 'test' ); // Array ( [0] => test )
```

Happy to remove if you still disagree.